wire-ios-request-strategy/Sources/Request Strategies/Feature configurations/GetFeatureConfigsActionHandlerTests.swift

```diff
@@ -136,6 +136,85 @@ class GetFeatureConfigsActionHandlerTests: MessagingTestBase {
 
         XCTAssertTrue(waitForAllGroupsToBeEmpty(withTimeout: 0.5))
     }
+    
+    func test_ItHandlesResponseWithoutMLS_200() throws {
+        syncMOC.performGroupedBlock {
+            // Given
+            let sut = GetFeatureConfigsActionHandler(context: self.syncMOC)
+            var action = GetFeatureConfigsAction()
+
+            // Expectation
+            let gotResult = self.expectation(description: "gotResult")
+
+            action.onResult { result in
+                switch result {
+                case .success:
+                    break
+
+                default:
+                    XCTFail("Expected 'success'")
+                }
+
+                gotResult.fulfill()
+            }
+
+            let payload = GetFeatureConfigsActionHandler.ResponsePayload(
+                appLock: .init(status: .enabled, config: .init(enforceAppLock: true, inactivityTimeoutSecs: 11)),
+                classifiedDomains: .init(status: .enabled, config: .init(domains: ["foo"])),
+                conferenceCalling: .init(status: .enabled),
+                conversationGuestLinks: .init(status: .enabled),
+                digitalSignatures: .init(status: .enabled),
+                fileSharing: .init(status: .enabled),
+                mls: nil,
+                selfDeletingMessages: .init(status: .enabled, config: .init(enforcedTimeoutSeconds: 22))
+            )
+
+            guard let payloadData = try? JSONEncoder().encode(payload) else {
+                XCTFail("failed to encode payload")
+                return
+            }
+
+            let payloadString = String(data: payloadData, encoding: .utf8)!
```

Comment:
move to the guard, prefer failing test than crashing

Reply:
The other test was already there, I've copied the other one and changed the mls in the payload to nil.
I'll fix the force unwrap in both of them then.